media-libs/audiofile/audiofile-0.3.6-r5.ebuild

```diff
@@ -32,6 +32,10 @@ src_prepare() {
 }
 
 multilib_src_configure() {
+	# Bug 914349
+	if tc-is-clang; then
+		append-flags "-stdlib=libstdc++"
```

Comment:
We can't do this - it breaks ABI if it was previously built with libcxx.

Reply:
Ah I see, what do you recommend I look at instead?

Reply:
Maybe it should be forced to build with gcc like how btop does?
https://gitweb.gentoo.org/repo/gentoo.git/tree/sys-process/btop/btop-1.2.13.ebuild#n20